app/controllers/health/team_patients_controller.rb

```diff
@@ -51,15 +51,28 @@ def index
       @direction = params[:direction]&.to_sym || :asc
       respond_to do |format|
         format.html do
-          medicaid_ids = @patients.map(&:medicaid_id)
-          @patients = patient_source.where(id: @patients.pluck(:id))
+          patient_ids = @patients.pluck(:id)
+          medicaid_ids = @patients.pluck(:medicaid_id)
```

Comment:
Probably isn't a huge deal, but we could pluck both of these in the same call to save a 
```suggestion
          ids = @patients.pluck(:id, :medicaid_id)
          patient_ids = ids.map(&:first)
          medicaid_ids = ids.map(&:last)
```